source/extensions/filters/http/jwt_authn/authenticator.cc

```diff
@@ -80,10 +83,9 @@ class AuthenticatorImpl : public Logger::Loggable<Logger::Id::jwt>,
   std::vector<JwtLocationConstPtr> tokens_;
   JwtLocationConstPtr curr_token_;
   // The JWT object.
-  std::unique_ptr<::google::jwt_verify::Jwt> jwt_;
+  std::unique_ptr<::google::jwt_verify::Jwt> owned_jwt_;
```

Comment:
s/owned_jwt/new_jwt/   as opposite to cached_jwt.

Reply:
Done.